src/server.ts

```diff
@@ -17,6 +17,7 @@ server.get('/*', (_req, res) =>
 
 export async function start() {
 	await meetbotManager.init();
+	await meetbotManager.scheduleBotsForMeetings();
```

Comment:
why not call this inside the init function ? Running a bunch of functions to enable stuff seems weird...we could pass an object to init with functionality we want to enable (such as calendar joins) which then run `scheduleBotsForMeetings`

Reply:
I like that idea. Cleaner